README.md

```diff
@@ -116,6 +116,9 @@ $ yarn truffle migrate --network YOUR_NETWORK_ID
 
 For more details see [Truffle's Documentation.](https://truffleframework.com/docs/truffle/getting-started/running-migrations)
 
+##Storage
+Sojourn uses IPFS to securely store each document, in a way where the user can access it from any device without compromising the security of their data. The way we do this is via Shamir's Secret Sharing Algorithm. Specically, it encrypt each file with AES, splits it apart with Shamir's Secret Sharing Algorithm, and spreads the shards onto IPFS. Web3-studio created an open source project integrating this pipeline into React-Native. It's code can be viewed [here](https://github.com/ConsenSys/react-native-sssa). Pull request are welcome and encouraged.
```

Comment:
```suggestion
Sojourn uses IPFS to securely store each document in a way where the user can access it from any device without compromising the security of their data. The way we do this is via Shamir's Secret Sharing Algorithm. Specifically, it encrypt each file with AES, splits it apart with Shamir's Secret Sharing Algorithm, and spreads the shards onto IPFS. Web3Studio created an open source project integrating this pipeline into React-Native, [`react-native-sssa`](https://github.com/ConsenSys/react-native-sssa). Pull request are welcome and encouraged.
```

The diff doesn't seem to be good, but It's
1. Spelling on Specifically
2. branding for Web3Studio
3. Less important, but I prefer making sentences read correctly and adding links to verbs and nouns rather than say 'click here'.